pyodata/v2/service.py

```diff
@@ -1581,10 +1590,11 @@ def function_import_handler(fimport, response):
 class Service:
     """OData service"""
 
-    def __init__(self, url, schema, connection):
+    def __init__(self, url, schema, connection, retain_null=False):
```

Comment:
I do not particularly like this API. generic parameters, followed by boolean flag.. it is not much future-proof, when need arise for fifth or sixth configuration parameter in service. 

I understand that this is getting quite entangled an messy, because Config class is in the model.py module and so far was not used by service.py at at. And it should in fact be in the root next to client or exceptions, definitely above pyodata/v2, simply because same config parameters should work IMHO in future v4 support as well. Perhaps such move will be in the 2.0.0 version, where backward compatibility can be broken.. 

But service.py is already at the moment depending on model.py: https://github.com/SAP/python-pyodata/blob/0599db299cf5823f810b7806c2a8dda1227dde6e/pyodata/v2/service.py#L20

So why not add the `config=None` to Service, instead of  `retain_null=False`.. and in client.py:72 just pass the same value as in client.py:68

https://github.com/SAP/python-pyodata/blob/0599db299cf5823f810b7806c2a8dda1227dde6e/pyodata/client.py#L68